src/view/breadcrumbs.component.tsx

```diff
@@ -30,6 +31,22 @@ const Breadcrumbs = (props: BreadcrumbsProps) => {
     navigateHomeAriaLabel,
   } = props;
 
+
+  const getTooltipFromAria = (navigateHomeAriaLabel: string): string => {
+    const lowerAriaLabel = navigateHomeAriaLabel.toLowerCase();
+
+    if (lowerAriaLabel.includes('catalogue')) {
+      return 'Catalogue Home';
+    } else if (lowerAriaLabel.includes('system')) {
+      return 'Systems Home';
+    } else if (lowerAriaLabel.includes('admin')) {
+      return 'Admin Home';
+    } else if (lowerAriaLabel.includes('manufacturer')) {
+      return 'Manufacturers Home'
+    } 
+    return 'Home';
+  }
```

Comment:
It would be better to change the navigateHomeAriaLabel prop in the breadcrumb component to homeLocation: 'System' | 'Admin' | .... We can then use homeLocation to construct both the aria-label and the tooltip title. This approach will ensure that the aria-labels are more consistent.